langchain4j/src/main/java/dev/langchain4j/data/document/source/ClassPathSource.java

```diff
@@ -0,0 +1,97 @@
+package dev.langchain4j.data.document.source;
+
+import static dev.langchain4j.data.document.Document.FILE_NAME;
+import static dev.langchain4j.data.document.Document.URL;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+import dev.langchain4j.data.document.DocumentSource;
+import dev.langchain4j.data.document.Metadata;
+
+/**
+ * Specialization of a {@link DocumentSource} that knows how to read from the classpath
+ */
```

Comment:
nit: `@see ClassPathDocumentLoader`

Reply:
I'm not sure what the comment `@see ClassPathDocumentLoader` is supposed to reference here? There is no reference to the word "folder" in this class...

Reply:
Sorry, I meant to create a back link from source to the loader. There is a javadoc link from loader to the source, but not vice versa